core-blocks/gallery/editor.scss

```diff
@@ -34,6 +34,16 @@
 		width: calc( 100% - 8px );
 		left: 4px;
 		margin-top: -4px;
+
+		// if this inline toolbar has negative margin, it's hidden by the overflow that we need for scrolling long captions
```

Comment:
Serious nitpick but I think it's good for comments to be full sentences and start with capital letters.

Reply:
Also, this comment is a bit indirect in saying what the objective/reason for this CSS is. How about: "Override negative margins so this toolbar isn't hidden by overflow. Overflow is needed for long captions."

Reply:
I dig serious nitpicks! 👍 👍